_api-reference/document-apis/bulk.md

```diff
@@ -35,7 +34,7 @@ POST _bulk
 POST <index>/_bulk
 ```
 
-Specifying the index in the path means you don't need to include it in the [request body]({{site.url}}{{site.baseurl}}/opensearch/rest-api/document-apis/bulk/#request-body).
+Specifying the index in the path means you don't need to include it in the [request body]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/#request-body).
```

Comment:
Capitalize the linked title:

`[Request body]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/#request-body)`

Reply:
Would need to rewrite the sentence with a learn more link.   "learn more, see [Proper link title]."
because it wouldn't be correct to have uppercase in the middle of the sentence.